docs/resources/azapi_resource.md

```diff
@@ -124,6 +124,8 @@ The following arguments are supported:
 }
 ```
 
+* `replace_triggered_by` = (Optional) Will trigger a replace of the resource when the value changes and is not `null`. This can be used by practitioners to force a replace of the resource when certain values change, e.g. changing the SKU of a virtual machine.
```

Comment:
I'm wondering if we can expand on this or docs elsewhere to provide an example of usage? It might not be clear to consumers how to practically use this attribute.

E.g if setting a property in the scheme from a var and you want that to trigger a change, then also specify the var in this dynamic.